packages/nimble-components/src/anchor/index.ts

```diff
@@ -35,6 +35,14 @@ export class Anchor extends AnchorBase {
     @attr
     public appearance: AnchorAppearance;
 
+    /**
+     * @public
+     * @remarks
+     * HTML Attribute: tabindex
+     */
+    @attr({ attribute: 'tabindex', converter: nullableNumberConverter })
+    public override tabIndex!: number;
```

Comment:
Double checking, do we need to update Angular / Blazor integrations to expose tabIndex on these components or is it available already because it's a native attribute?

Reply:
From my experimentation, it works without adding anything to our Angular/Blazor wrappers.